jabbercat/models.py

```diff
@@ -1,28 +1,27 @@
 import bisect
-import collections.abc
 import enum
 import typing
 import unicodedata
-
-import lxml.builder
-import lxml.etree
+from itertools import chain
+from typing import List
 
 import aioxmpp.callbacks
 import aioxmpp.structs
-
+import jabbercat.avatar
 import jclib.conversation
 import jclib.identity
 import jclib.instrumentable_list
 import jclib.metadata
 import jclib.roster
 import jclib.utils
+import lxml.builder
+import lxml.etree
+from aioxmpp.callbacks import Signal
+from jclib.roster import AbstractRosterItem
 
-import jabbercat.avatar
 import jabbercat.utils
-
 from . import Qt, model_adaptor, utils
 
-
```

Comment:
Leave two blank lines between imports and code.

Reply:
OK